ignite/templates/app/files/app/ibc.go.plush

```diff
@@ -38,7 +38,7 @@ import (
 )
 
 // registerIBCModules register IBC keepers and non dependency inject modules.
-func (app *App) registerIBCModules() {
+func (app *App) registerIBCModules(appOpts servertypes.AppOptions) error {
```

Comment:
Maybe, until the argument is implemented:

```suggestion
func (app *App) registerIBCModules(_ servertypes.AppOptions) error {
```

Reply:
the code we will inject into the placeholder in this function needs this var